src/components/Image/Image.js

```diff
@@ -81,11 +93,28 @@ export const Image = (props) => {
 
   const { src, alt, srcset, thumbnail, ...otherImageProps } = props.image;
 
+  // This block checks whether the user specified fallback image actually works or not
+  const [isLoading, setIsLoading] = useState(true);
+  const [isValidFallback, setIsValidFallback] = useState(false);
+  useEffect(() => {
+    fetch(props.placeholderImg).then(res => {
+      setIsValidFallback(res.status === 200);
+      setIsLoading(false);
+    }).catch((_reason) => {
+      setIsLoading(false);
+    })
+  }, []);
```

Comment:
Here is an error message:
`ESLint: React Hook useEffect has a missing dependency: 'props.placeholderImg'. Either include it or remove the dependency array.(react-hooks/exhaustive-deps)`

Reply:
Done.